nx/lib/nx.ex

```diff
@@ -2762,23 +2762,41 @@ defmodule Nx do
         ]
       >
 
+  ## Vectorized tensors
```

Comment:
Perhaps this is a bit too late but, once we add such new sections, it would be nice to have a succinct description of how it behaves. For example:

```suggestion
  ## Vectorized tensors

  The new axis is added after the vectorized axes:

```